tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -99,50 +99,51 @@ def task(
         "restore_destination": restore_destination,
         "source_bucket": bucket_name,
     }
-    # post to recovery queue. Retry using exponential delay if it fails
+
+        # post to DB-queue. Retry using exponential delay if it fails
     for retry in range(max_retries+1):
         try:
-            shared_recovery.post_entry_to_queue(
-                new_data, shared_recovery.RequestMethod.NEW_JOB, recovery_queue_url
+            shared_recovery.update_status_for_file(
+                job_id,
+                granule_id,
+                filename,
+                shared_recovery.OrcaStatus.STAGED,
+                None,
+                db_queue_url,
             )
+            break
         except Exception as ex:
             LOGGER.error(
-                f"Ran into error posting to SQS {recovery_queue_url} {retry+1} time(s) with exception {ex}"
+                f"Ran into error posting to SQS {db_queue_url} {retry+1} time(s) with exception {ex}"
             )
             my_base_delay = exponential_delay(my_base_delay, retry_backoff)
             continue
-        break
     else:
-        message = f"Error sending message to {recovery_queue_url} for {new_data}"
+        message = f"Error sending message to {db_queue_url} for {new_data}"
         logging.critical(message)
         raise Exception(message)
 
     # resetting my_base_delay
     my_base_delay = retry_sleep_secs
-    # post to DB-queue. Retry using exponential delay if it fails
+
+    # post to recovery queue. Retry using exponential delay if it fails
     for retry in range(max_retries+1):
         try:
-            shared_recovery.update_status_for_file(
-                job_id,
-                granule_id,
-                filename,
-                shared_recovery.OrcaStatus.STAGED,
-                None,
-                db_queue_url,
+            shared_recovery.post_entry_to_queue(
+                new_data, shared_recovery.RequestMethod.NEW_JOB, recovery_queue_url
             )
+            break
         except Exception as ex:
             LOGGER.error(
-                f"Ran into error posting to SQS {db_queue_url} {retry+1} time(s) with exception {ex}"
+                f"Ran into error posting to SQS {recovery_queue_url} {retry+1} time(s) with exception {ex}"
             )
             my_base_delay = exponential_delay(my_base_delay, retry_backoff)
             continue
-        break
     else:
-        message = f"Error sending message to {db_queue_url} for {new_data}"
+        message = f"Error sending message to {recovery_queue_url} for {new_data}"
         logging.critical(message)
```

Comment:
Shouldn't this be LOGGER.critical instead of logging.critical?

Reply:
changed to using cumulus logger